src/lib/metrics.js

```diff
@@ -141,6 +142,14 @@ const prometheus = (reqType, dataSet, username, client, groups, duration) => {
     help: `Request time of requests from members of the ${g} group`
   }));
 
+  if(blocked) {
+    prometheusMetrics.req[dataSet].blocked = prometheusMetrics.req[dataSet].blocked || new promClient.Counter({
+      name: `${prometheusPrefix}_req_${escape(dataSet)}_blocked`,
+      help: `Number of blocked records for ${dataSet}`
+    });
+    prometheusMetrics.req[dataSet].blocked.inc();
+  }
```

Comment:
This (including the if statement) should be moved below the 'Update metrics' comment below.